sv_pipeline/exome/load_data.py

```diff
@@ -405,7 +405,7 @@ def main():
     p.add_argument('--sample-type', default='WES')
     p.add_argument('--es-host', default='localhost')
     p.add_argument('--es-port', default='9200')
-    p.add_argument('--num-shards', default=6)
+    p.add_argument('--num-shards', default=1)
```

Comment:
L349 has `num_shards=6`; I think since there's a default for the argument, that the function should not default to 6, but should we change that as well?